src/common-components/search-bar/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
```

Comment:
the code for search should've been in a separate branch and not here in nav for merging, let's not mix things if they're not finished yet.